mozetl/graphics/create_cluster.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
Are these scripts for testing? It's not clear, some comment about this would be nice.

Reply:
Scripts are for testing, I'll add a comment.